### DISCUSSION TITLE
I've scanned but no movies are showing?

### DISCUSSION THREAD

**Discussion by user50237** [stashapp/stash#3345] [Q&A] [answered]
I went to Library in Settings and added a directory of the video folder I want to import into Stash then went to Tasks to do a scan but no videos are showing in the Movies tab.

I have videos formats of all different kinds (.MP4, .MKV etc.) and still no videos show up, images work fine.

**Comment by MrX292** [comment]
did you check the Scenes Tab?
all videos in stash are Scenes movies are either manually created or trough a scraper.

**Comment by user50237** [comment]
> did you check the Scenes Tab?
> all videos in stash are Scenes movies are either manually created or trough a scraper.

Nothing in Scenes tab too

**Comment by HereIam80** [comment]
What does the log say? On what OS are you on? Is this a new install?
This could be handled much faster in the discord btw.

**Comment by philpw99** [comment]
Are you using Windows ?
The Stash will not show you the debug log unless you open a console and run "stash-win.exe" inside that console.
From there you will be able to see the error messages.

**Reply by user50237** [reply]
I'm using Windows 10, accessing through http://localhost:9999/ and my library directory is on an external hard drive. Here are the 2 frequent errors that show when running the scan again (I changed folder and file names to be SFW). If it matters the Stash installation files and ffmpeg which I'm not sure installed correctly are on my desktop (I think installed to my desktop) and I don't know where to put them and how to move them safely without breaking the install.

```
Error: exec: "ffprobe.exe": cannot run executable found relative to current directory
2023-01-07 12:25:11Info    Calculating fingerprints for Z:\foldername\foldername2\IMG_0158.MOV.webm ...
2023-01-07 12:25:11Error   error processing "Z:\\foldername\\foldername2\\0617106648332.webm": running ffprobe on "Z:\\foldername\\foldername2\\1617106648332.webm": FFProbe encountered an error with <Z:\foldername\foldername2\0617106648330.webm>.
Error JSON:

23-01-07 12:32:32Error   error processing "Z:\\foldername\\foldername2\\foldername3\\1080p_60fps_VP9-128kbit_AAC.mkv": running ffprobe on "Z:\\foldername\\foldername2\\foldername3\\1080p_60fps_VP9-128kbit_AAC.mkv": FFProbe encountered an error with <Z:\foldername\foldername2\foldername3\1080p_60fps_VP9-128kbit_AAC.mkv>.
Error JSON:
```

**Reply by HereIam80** [reply, marked as answer]
You seem to experience this bug: #3304 .
Either change the paths to its default path which is `C:\users\USER\.stash` or use the 0.17 version of stash.
